setup.py

```diff
@@ -17,15 +17,15 @@
 
 setup(
     name='awsiot',
-    version='2.0',
+    version='2.2',
     description='AWS IoT SDK based on the AWS Common Runtime',
     author='AWS SDK Common Runtime Team',
     url='https://github.com/awslabs/aws-iot-device-sdk-python-v2',
     packages = find_packages(),
     install_requires=[
-        'aws_crt',
+        'awscrt==0.2.14',
```

Comment:
Don't want to do >=?

Reply:
no, because we haven't determined what the ABI line is yet and for now we want to do explicit versioning until we do.